### PR TITLE
Add server for deploy on heroku

### PR DESCRIPTION
**Pull request by shuliakleonid**

### CODE REVIEW

File: exadel-trainee-app/src/app/services/store/auth-store.service.ts [hunk 1]
```diff
@@ -51,6 +58,21 @@ export class AuthStoreService {
 
   signOut(): void {
     this.user = this.initialState;
+    this.setToken(null);
+    localStorage.clear();
     this.router.navigate(['/']);
   }
+
+  private setToken(response: UserResponseType | null) {
```
Comment (RomPavel) [inline exadel-trainee-app/src/app/services/store/auth-store.service.ts:66]: Lets move this method to data-store-service.
Its better to work with localStorage in one place

File: exadel-trainee-app/src/app/services/store/auth-store.service.ts [hunk 2]
```diff
@@ -24,20 +25,26 @@ export class AuthStoreService {
     private readonly authHttpService: AuthHttpService,
     private readonly errorStoreService: ErrorStoreService,
     private readonly router: Router,
-  ) {}
+    private dataUser: DataStoreService,
+  ) {
+    console.log(this.dataUser, 'dataUser');
```
Comment (RomPavel) [inline exadel-trainee-app/src/app/services/store/auth-store.service.ts:30]: remove this

File: exadel-trainee-app/src/app/services/store/auth-store.service.ts [hunk 3]
```diff
@@ -51,6 +58,21 @@ export class AuthStoreService {
 
   signOut(): void {
     this.user = this.initialState;
+    this.setToken(null);
+    localStorage.clear();
```
Comment (RomPavel) [inline exadel-trainee-app/src/app/services/store/auth-store.service.ts:62]: remove this

File: exadel-trainee-app/src/app/services/store/auth-store.service.ts [hunk 4]
```diff
@@ -51,6 +58,21 @@ export class AuthStoreService {
 
   signOut(): void {
     this.user = this.initialState;
```
Comment (RomPavel) [inline exadel-trainee-app/src/app/services/store/auth-store.service.ts:60]: this logic feels messy
you have userA in localStorage, then you signIn by userB and when you signOut initalState still contains userA, so we return back to userA

**Comment by RomPavel** [comment]
Do you plan to merge it to master? So we will need to pull master into dev after?

**Comment by shuliakleonid** [comment]
> Do you plan to merge it to master? So we will need to pull master into dev after?

I want check deploy settings? because I`m not sure in it. Check my be only in master branch.

**Review by RomPavel — APPROVED** [review]

**Pull request merged**